### ISSUE TITLE
x-www-form-urlencoded support?

### ISSUE THREAD

**Issue by FlorianWalther** [typicode/json-server#876]
Hello,
does this API support x-www-form-urlencoded requests?

**Comment by b4dnewz** [comment]
yes it does, [source](https://github.com/typicode/json-server/blob/master/src/server/body-parser.js#L5)
